editor/script/room.js

```diff
@@ -195,6 +197,10 @@ function RoomTool(canvas) {
 		isDragDeletingTiles = false;
 	}
 
+	function clamp(val, min, max){
```

Comment:
hmm looks like the clamp changes from your other review got mixed in here - do you mind removing them? probably you just need to sync again

Reply:
I couldn't find that function definition in my local repo so I just pushed again and it seems to have pushed some changes. Hopefully that fixed it.

Reply:
there's still something weird about the changes it's showing, but I'm going to just try to merge it and cross my fingers! thanks for the patience :)